frontend/src/services/fileUploadService.js

```diff
@@ -4,5 +4,5 @@ export const fileUpload = (file) => {
     const formData = new FormData();
     formData.append('file', file);
 
-    return axios.post("/api/upload", formData);
+    return axios.post("/api/upload", formData).then((response) => response.data);
```

Comment:
Is there a `catch` at the point where the response is used?

Reply:
This pertains to your other point in the FileUpload.js , I have not yet decided WHAT should happen in this case so this will be done once the MVP is done.